api/jobs.yaml

```diff
@@ -573,13 +573,25 @@ definitions:
     required:
       - failure
     properties:
+      taskName:
+        type: string
+        description: The name of the task that generated the failure message
       failure:
         type: string
         description: The failure message
       timestamp:
         type: string
         format: date-time
         description: The time at which this failure occurred
+      stdout:
+        type: string
+        description: Path to the standard output file for this task or shard
+      stderr:
+        type: string
+        description: Path to the standard error file for this task or shard
+      callRoot:
```

Comment:
This language might be obvious for Cromwell users, but in terms of the general jobs API I have no idea what a "callRoot" is and what I should expect to find here. Is there any way we can encode this information more generally e.g. by renaming this?

Reply:
"callRoot" was introduced to `jobs.yaml` in #271 (see https://github.com/DataBiosphere/job-manager/blob/master/api/jobs.yaml#L558)